x/gov/msgs.go

```diff
@@ -41,10 +44,10 @@ func (msg MsgSubmitProposal) Type() string  { return TypeMsgSubmitProposal }
 
 // Implements Msg.
 func (msg MsgSubmitProposal) ValidateBasic() sdk.Error {
-	if len(msg.Title) == 0 {
+	if len(msg.Title) == 0 || len(msg.Title) > MaxTitleLength {
```

Comment:
can we have another `if` validation for title and description with a message that exceeds max length ?

Reply:
for better UX